packages/manager/modules/pci/src/components/project/images-list/images-list.service.js

```diff
@@ -3,12 +3,71 @@ import groupBy from 'lodash/groupBy';
 import map from 'lodash/map';
 import pick from 'lodash/pick';
 import omit from 'lodash/omit';
+import snakeCase from 'lodash/snakeCase';
 
 import Image from './images.class';
 
+function getDistribution(name, type) {
+  const regex = {
+    linux: [
+      {
+        name: 'ubuntu',
+        regex: /^Ubuntu/i,
+      },
+      {
+        name: 'freebsd',
+        regex: /^FreeBSD/i,
+      },
+      {
+        name: 'coreos',
+        regex: /^CoreOS/i,
+      },
+      {
+        name: 'debian',
+        regex: /^Debian/i,
+      },
+      {
+        name: 'centos',
+        regex: /^Cent[\s-]?OS/i,
+      },
+      {
+        name: 'fedora',
+        regex: /^Fedora/i,
+      },
+      {
+        name: 'dokku',
+        regex: /^Dokku/i,
+      },
+    ],
+    windows: [
+      {
+        name: 'windows_server_2012',
+        regex: /^Win[a-zA-Z\s-]+2012/i,
+      },
+      {
+        name: 'windows_server_2016',
+        regex: /^Win[a-zA-Z\s-]+2016/i,
+      },
+    ],
+  };
+
+  if (regex[type]) {
+    for (let i = 0, l = regex[type].length; i < l; i += 1) {
+      if (regex[type][i].regex.test(name)) {
+        return regex[type][i].name;
+      }
+    }
+
+    return ''.concat(type, '_other');
+  }
+
+  return 'unknown';
```

Comment:
is it required to translate this? is it shown on UI?

Reply:
No, it's not required.